css/main.css

```diff
@@ -0,0 +1,1623 @@
+  height: 200px;
+  width: 90%;
+  margin-bottom: 10px;
+}
+
+.section1-2-heading {
+  font-weight: 700;
+  letter-spacing: -0.84px;
+  color: #fff;
+  font-family: Georgia, 'Times New Roman', Times, serif;
+  gap: normal;
+  position: relative;
+  top: 88px;
+  margin-left: 6px;
+  line-height: 20px;
+  margin-top: 50px;
+  margin-bottom: 30px;
+}
+
+.section1-2-heading-title-link {
+  text-decoration: none;
+  color: #fff;
+  position: relative;
+  top: 130px;
+  font-weight: lighter;
+  font-size: 20px;
+  font-family: Georgia, 'Times New Roman', Times, serif;
+  margin-left: 6px;
+}
+
+.media-align-text {
+  text-align: left;
+}
+
+#ces-move {
+  margin-left: 6px;
+  position: relative;
+  top: 42px;
+}
+
+.section-1-3 {
+  background: url("https://cdn0.tnwcdn.com/wp-content/blogs.dir/1/files/2020/07/deepfake-ai-voice-celebrities-politicians-796x417.jpg");
+  background-size: cover;
+  background-repeat: no-repeat;
+  height: 200px;
+  width: 90%;
+  font-size: 20px;
+  line-height: 20px;
+  margin-top: 0;
+  margin-bottom: 30px;
+}
+
+.section1-3-heading {
+  font-weight: 700;
+  letter-spacing: -0.84px;
+  color: #fff;
+  font-family: Georgia, 'Times New Roman', Times, serif;
+  gap: normal;
+  position: relative;
+  top: 32px;
+  margin-left: 6px;
+  line-height: 20px;
+  margin-top: 50px;
+  margin-bottom: 30px;
+}
+
+.media-align-2 {
+  margin-top: 0;
+}
+
+.media-align-3 {
+  margin-top: 0;
+}
+
+.section1-3-heading-title-link {
+  text-decoration: none;
+  color: #fff;
+  position: relative;
+  top: 75px;
+  font-weight: lighter;
+  font-size: 20px;
+  font-family: Georgia, 'Times New Roman', Times, serif;
+  margin-left: 6px;
+}
+
+#ai {
+  margin-left: 6px;
+  position: relative;
+  bottom: 10px;
+}
+
+.section1-grid {
+  display: grid;
+  grid-template-columns: 1fr;
+  grid-template-rows: 1fr 1fr;
+}
+
+#tnw-logo {
+  width: 20%;
+  height: 10%;
+  position: relative;
+  margin: 9px;
+  top: -75px;
+}
+
+.main-container {
+  margin: 10px 0 0 0;
+  width: auto;
+  height: auto;
+}
+
+.link {
+  text-decoration: none;
+  color: rgb(255, 86, 34);
+  font-size: 20px;
+  font-weight: 700;
+}
+
+.link-2 {
+  text-decoration: none;
+  color: rgb(255, 86, 34);
+  font-size: 14px;
+  height: 31px;
+  margin-top: 30px;
+  margin-right: 23px;
+  margin-bottom: 0;
+}
+
+.f1-grid {
+  display: block;
+}
+
+.f1 {
+  width: 100%;
+  height: 100px;
+  line-height: 0;
+}
+
+.f1-img {
+  width: 75px;
+  height: 75px;
+}
+
+.f1-media {
+  margin-left: 100px;
+  margin-top: -80px;
+}
+
+.f1 h4 {
+  margin: 0 0 0 8px;
+}
+
+.f1-flex {
+  display: flex;
+  margin: 10px 0 0 8px;
+}
+
+.f1-by {
+  text-decoration: none;
+  color: rgb(85, 85, 85);
+  font-size: 8px;
+}
+
+.f1-flex span {
+  color: rgb(85, 85, 85);
+  font-size: 8px;
+}
+
+.gallery-link {
+  text-decoration: none;
+  color: black;
+  font-weight: 700;
+  font-size: 10px;
+  text-align: left;
+  line-height: 20px;
+}
+
+.flex {
+  display: block;
+  justify-content: space-between;
+}
```

Comment:
- [ ] To fix this in **mobile**, just use
```
.flex {
    display: flex;
    justify-content: space-between;
    flex-direction: column;
}
```
and then take out the margin-top:30px of the `<a>'s`
![image](https://user-images.githubusercontent.com/53256139/89096561-17bfe680-d39d-11ea-98a9-29f042aa58d5.png)